x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -15,301 +15,276 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
   {
     key: 'linux.advanced.agent.connection_delay',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'How long to wait for agent connectivity before sending first policy reply, in seconds. Default: 60.',
   },
   {
     key: 'linux.advanced.artifacts.global.base_url',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Base URL from which to download global artifact manifests. Default: https://artifacts.security.elastic.co.',
   },
   {
     key: 'linux.advanced.artifacts.global.manifest_relative_url',
     first_supported_version: '7.11',
-    documentation: '',
-  },
-  {
-    key: 'linux.advanced.artifacts.global.ca_cert',
-    first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Relative URL from which to download global artifact manifests. Default: /downloads/endpoint/manifest/artifacts-<version>.zip.',
```

Comment:
@gabriellandau I was wondering about the best way to ensure that code changes translate to doc changes, but I'm not sure I understand your suggestion. Are you saying that by also adding the defaults to the endpoint config file that it will be easier to remember to update the schema file when updating the code and config file?

Reply:
I'm making a blocker issue for every release that someone has to look at this file and make sure it is right. There's no good solution otherwise. Kibana is public and Endpoint is private. Not all Endpoint advanced options are being exposed in Kibana. Endpoint's defaults are hard coded in C++. Etc.

Reply:
Dan gets it.  Another approach would to be to have every user-facing advanced option be mandatory (or else face policy failure), and have Kibana send every one.  This would entail removing every hard-coded C++ default that can be configured via Kibana.

Just a thought.